Assets/Scripts/PlayerManager.cs

```diff
@@ -13,8 +13,9 @@ public class PlayerCameraBlendingOptions
 public class PlayerManager : MonoBehaviour
 {
     [Header("Stamina")]
-    public float staminaIncrease = 0.1f;
-    public float staminaDecrease = 0.001f;
+    public float staminaIncrease = 0.01f;
+    public float staminaDecrease = 0.005f;
```

Comment:
Probably should rename these to being awakeness-related now
lets pretend stamina was never a thing LOL